site/e2e/tests/signOut.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { test, expect } from "@playwright/test";
+import { accessibleDropdownLabel } from "modules/dashboard/Navbar/UserDropdown/UserDropdown";
+import { Language } from "modules/dashboard/Navbar/UserDropdown/UserDropdownContent";
+import * as constants from "../constants";
+import { assertNoUncaughtRuntimeError } from "../helpers";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("Signing in and out", async ({ page, baseURL }) => {
+  const handleSignIn = async () => {
```

Comment:
I wasn't sure if this was really necessary, but I couldn't get to the meat of the sign out tests until I had the UI sign in